scripts/base/frameworks/logging/main.zeek

```diff
@@ -373,6 +373,26 @@ export {
 		## New Filters created for this stream will inherit
 		## this policy hook, unless they provide their own.
 		policy: PolicyHook &optional;
+
+		## Event groups associated with this stream that are disabled
+		## when :zeek:see:`Log::disable_stream` is invoked and
+		## re-enabled during :zeek:see:`Log::enable_stream`.
+		##
+		## This can be used to short-circuit event handlers that are
+		## solely responsible for logging functionality at runtime
+		## when a log stream is disabled.
+		##
+		## This field allows for both, attribute event groups and module
+		## event groups. If an attribute event group exists with a
+		## given name, it is always selected. Further, if the group
+		## name contains a ``::`` module separator and a corresponding
+		## module event group exist, is is used. This allows to toggle
+		## ``&group="DNS::Logging"`` and ``module DNS::Logging`` handlers
+		## by providing just ``DNS::Logging`` as group name.
+		##
+		## It is not possible to disable top-level module event groups
+		## like ``DNS``.
```

Comment:
I'm wondering if this is too inconsistent: seems from a user perspective, there's really no reason why this shouldn't work for top-level module names. Instead of looking for `::`, we could instead just check if a module event group exists of that name?

Reply:
I've removed the special case. If there's a module of a given name, it's now toggled.